CHANGES

```diff
@@ -7,6 +7,35 @@ or get the entire source code repository and view log history:
 $ git clone https://github.com/greenbone/gvm-libs.git
 $ cd gvm-libs && git log
 
+gvm-libs 1.0+beta2 (2018-12-04)
+
+This is the second beta release of the gvm-libs module 1.0 for the Greenbone
+Vulnerability Management (GVM) framework.
+
+Many thanks to everyone who has contributed to this release:
+Hani Benhabiles, Christian Fischer, Matt Mundell, Juan Jose Nicola,
+Timo Pollmeier, Jan-Oliver Wagner and Michael Wiegand.
+
+Main changes compared to gvm-libs 1.0+beta1:
+* Redis kb content has been switched to use sorted lists.
```

Comment:
Would rephrase it as: KB backend has been switched to use Redis lists instead of unsorted sets.